src/main/java/io/seqera/tower/cli/commands/runs/DumpCmd.java

```diff
@@ -112,7 +112,10 @@ protected Response exec() throws ApiException, IOException {
             tar.add("workflow-launch.json", collectWorkflowLaunch(wspId));
             tar.add("workflow-metrics.json", collectWorkflowMetrics(wspId));
             tar.add("workflow-tasks.json", collectWorkflowTasks(wspId));
-            tar.add("nextflow.log", collectNfLog(wspId));
+            var nfLog = collectNfLog(wspId);
+            if (nfLog != null) {
+                tar.add("nextflow.log", nfLog);
+            }
```

Comment:
Let's add more info for the user here
```suggestion
            iif (nfLog != null) {
                tar.add("nextflow.log", nfLog);
            } else {
                progress.println(ansi("\t- No data collected, skipping")); // nextflow-run workflows doesn't upload log
            }
```

Reply:
[1f5a5d6](https://github.com/seqeralabs/tower-cli/pull/520/commits/1f5a5d69d39022f2c08d1cfd0edc01dce0723bb3)